IMOMaritimeSingleWindow/Client/src/app/main-content/content-container/basis-data/user/register-user/register-user.component.ts

```diff
@@ -108,9 +112,10 @@ export class RegisterUserComponent implements OnInit, OnDestroy {
     this.getAllRolesSubscription.unsubscribe();
   }
   userExists(emailValid: boolean) {
+
     if (emailValid) {
 
-      if (this.newUser) {
+      if (this.user.email != this.preUpdateEmail) {
```

Comment:
The page still displays the wrong text for confirmButtonTitle when editing the different variables.
Changing role/email or organization displays the text = "Register user" instead of the intended "Apply changes" which may confuse the end user into thinking they are creating a new user instead of editing an existing one

Reply:
Need some validation for what can be inputed as a name (see picture)
This is a recurring problem in the client (as in not your fault so dont sweat it :) ), 
![image](https://user-images.githubusercontent.com/16537436/49744092-61c1f900-fc9c-11e8-8df2-0ae00bd003d1.png)

Reply:
Issues Resolved.